src/hooks/useWithdrawLiquidityHandler.tsx

```diff
@@ -34,13 +35,24 @@ export const useWithdrawLiquidityHandler = () => {
   }
 
   async function approveTransaction() {
-    // Deposit pool contract address
-    let address = process.env.NEXT_PUBLIC_DEPOSIT_POOL_ADDRESS
-    if (provider && address) {
-      if (accountInfo && accountInfo?.address) {
-        setdepPool(new ethers.Contract(address, DepPool.abi, provider.getSigner(accountInfo?.address)))
+    // Position Manager contract address
+    let address =
+      process.env.NEXT_PUBLIC_ENVIRONMENT === 'local'
+        ? process.env.NEXT_PUBLIC_POSITION_MANAGER_ADDRESS || ''
+        : '0x3eFadc5E507bbdA54dDb4C290cc3058DA8163152'
+    if (provider) {
+      if (process.env.NEXT_PUBLIC_ENVIRONMENT === 'local') {
+        if (accountInfo && accountInfo?.address) {
+          setPosManager(new ethers.Contract(address, PosManager.abi, provider.getSigner(accountInfo?.address)))
+        } else {
+          setPosManager(new ethers.Contract(address, PosManager.abi, provider))
+        }
       } else {
-        setdepPool(new ethers.Contract(address, DepPool.abi, provider))
+        if (accountInfo && accountInfo?.address) {
+          setdepPool(new ethers.Contract(address, DepPool.abi, provider.getSigner(accountInfo?.address)))
+        } else {
+          setdepPool(new ethers.Contract(address, DepPool.abi, provider))
+        }
       }
     } else {
       console.log('Please connect wallet')
```

Comment:
all `console.log()` statements should be converted into toasts. see [this gist](https://gist.github.com/Pochetes/37ac923536119df3ef77352ee38b9483) to see how I implemented it in another project.

Reply:
you can do this in another PR

Reply:
https://www.notion.so/All-Console-log-statements-should-be-converted-to-toasts-1c472d4fe9a740c4bf9cb55763a43772